### DISCUSSION TITLE
QoL enhancement to the interface

### DISCUSSION THREAD

**Discussion by Karnor** [slyh/Cities2-TrafficLightsEnhancement#27] [Ideas]
I love this mod, this and the Pedestrian patfinding mod added 30K to my capped city by freeing my simulator from needless recalculations for a while, it was like a mild OC to my cpu :)

But having gone round all my junctions to set the lanes, it became slightly tedious pretty quickly, with too many clicks needed to configure a junction.

Would it be possible to open and close/save all the lane configurations in one go, like a table?

So for example one icon above the road, which when clicked opens a table with one row per lane each with tick boxes for R(ight) L(eft) A(head), and cancel/reset/close buttons.

Then the save button for the junction saves both the phase and lane settings.

**Comment by slyh** [comment]
That's a really good suggestion, I will give it a try after rewriting the UI.
